packages/devtools_app/lib/src/screens/inspector_v2/inspector_tree_controller.dart

```diff
@@ -349,8 +349,12 @@ class InspectorTreeController extends DisposableController
 
   double get horizontalPadding => 10.0;
 
+  /// Returns the indentation of a row at the given [depth] in the inspector.
+  ///
+  /// This indentation corresponds to the center of the icon next to the widget
+  /// name.
   double getDepthIndent(int depth) {
-    return (depth + 1) * inspectorColumnWidth + horizontalPadding;
+    return (depth + 1) * inspectorColumnIndent + horizontalPadding;
```

Comment:
to improve the docs here, can you explain why we are doing depth + 1 ? I assume it is because depth is 0 based?

Reply:
Done! It is because it is 0 based